app/assets/javascripts/replace_search_text.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
+  var width = $(window).width();
+  if(width < 480 ){
+    $(".submit-search-text").text("");
```

Comment:
Because the search text takes up way too much space on a 320px wide screen, so I figure we should toggle the text off so we have more room for the actual search bar space.